packages/@lwc/integration-karma/test/template/directive-lwc-inner-html/index.spec.js

```diff
@@ -68,18 +68,16 @@ describe('type conversion', () => {
     });
 });
 
-it('applies styles to injected content', (done) => {
+it('applies styles to injected content', async () => {
     const elm = createElement('x-inner-html', { is: XInnerHtml });
     elm.content = '<b>Test</b>';
     document.body.appendChild(elm);
 
     // When running with synthetic shadow a micro task is needed to for the MutationObserver to add
     // the styling tokens. For IE11 specifically, we need to wait for a full task.
```

Comment:
```suggestion
    // the styling tokens.
```

We no longer support IE, so as you noticed, we can do a simple `Promise.resolve()` microtask instead of a `setTimeout`.